src/components/Rockets/Rockets.jsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable operator-linebreak */
+// import _ from 'lodash';
```

Comment:
- [ ] Remove this line of commented-out code if it is not being used by the component.